### PR TITLE
Improve README for Rails to reload app/bot/**/*.rb files

### PR DESCRIPTION
**Pull request by elct9620**
Hi, 

I try to let Rails auto reload files in `app/bot` and I found `FileUpdateChecker` in [grape](https://github.com/ruby-grape/grape#reloading-in-rails-applications) README.

I update README to use same way to let bot files can reload by Rails without restart server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,7 +312,15 @@ reference constants. You'll need to explicitly load `app/bot`, then:
 ```ruby
 # config/initializers/bot.rb
 unless Rails.env.production?
-  Dir["#{Rails.root}/app/bot/**/*.rb"].each { |file| require file }
+  api_files = Dir[Rails.root.join('app', 'bot', '**', '*.rb')]
```
Comment (gauravtiwari) [inline README.md]: @elct9620 Probably, we should use something like `bot_files` or `bots` instead or `api_files` in variables?

File: README.md [hunk 2]
```diff
@@ -312,7 +312,15 @@ reference constants. You'll need to explicitly load `app/bot`, then:
...
+  api_reloader = ActiveSupport::FileUpdateChecker.new(api_files) do
+    api_files.each{ |file| require_dependency file }
+  end
+  ActionDispatch::Callbacks.to_prepare do
+    api_reloader.execute_if_updated
```
Comment (gauravtiwari) [inline README.md]: bots_reloader may be?

File: README.md (hunk 1 above)
Comment (elct9620) [inline README.md]: Yeah, I think change it to `bot_files` will better than `api_files`

**Comment by elct9620** [comment]
@gauravtiwari I have updated README code from `api_files` into `bot_files`

**Comment by gauravtiwari** [comment]
@elct9620 Awesome thanks! Should we add this too above the section you have added to make it more clear? 

```ruby
# Auto-load bots and its subdirectories
config.paths.add File.join('app', 'bots'), glob: File.join('**', '*.rb')
config.autoload_paths += Dir[Rails.root.join('app', 'bots', '*')]
```

**Comment by elct9620** [comment]
@gauravtiwari Yeah, I also add it into my `config/application.rb`. But if I only add it, the files still not reload because there is no class need to reload which detected by Rails.

**Comment by gauravtiwari** [comment]
Oh yeah! that's right @elct9620 I guess this should be it then 👍

**Comment by elct9620** [comment]
@gauravtiwari I update README add code into README, but I am not sure is my description good for this code snippet.

File: README.md [hunk 3]
```diff
@@ -312,10 +312,26 @@ reference constants. You'll need to explicitly load `app/bot`, then:
 ```ruby
 # config/initializers/bot.rb
 unless Rails.env.production?
-  Dir["#{Rails.root}/app/bot/**/*.rb"].each { |file| require file }
+  bot_files = Dir[Rails.root.join('app', 'bot', '**', '*.rb')]
```
Comment (gauravtiwari) [inline README.md]: @elct9620 One small change, just rename `bot` to `bots` so, it matches with the `application.rb` folder.

**Comment by elct9620** [comment]
@gauravtiwari Ok, I updated it.

**Comment by gauravtiwari** [comment]
@elct9620 Awesome stuff! Have some 🍰

**Comment by dmitryshagin** [comment]
Would you, guyes consider releasing v0.9?

**Comment by jgorset** [comment]
Absolutely, @dmitryshagin! Released in 530db0cb809f88dc7979f9e5d657f716984226c7.

**Comment by gauravtiwari** [comment]
🎉  🍰  @jgorset

**Pull request merged**